docs/automated-workflows/dagman-introduction.rst

```diff
@@ -69,27 +69,34 @@ four node DAG would be described as the following in the DAG description file:
 
     # File name: diamond.dag
 
-    JOB A A.sub
-    JOB B B.sub
-    JOB C C.sub
-    JOB D D.sub
+    NODE A A.sub
+    NODE B B.sub
+    NODE C C.sub
+    NODE D D.sub
     PARENT A CHILD B C
     PARENT B C CHILD D
 
-.. _DAGMan JOB:
+.. _DAGMan NODE:
 
-JOB
----
+NODE
+----
 
-The :dag-cmd:`JOB[Usage]` command specifies a list of one or more HTCondor jobs that
-become the core of a node in the DAG. The syntax used for each :dag-cmd:`JOB`
+.. sidebar:: Old JOB command
+
+    The previous command ``JOB`` is a synonym of the ``NODE`` command.
+    Both create a node for in a DAG the same way. The ``JOB`` command
+    can still be specified for backwards compatibility reasons, but it
+    is preferred to used the ``NODE`` keyword instead.
```

Comment:
I might add a "why" here -- something like "This is because a dagman node can contain a list of jobs, and some users found it confusing that a list of jobs was represented by an object named "job".